bugsnag/client.py

```diff
@@ -208,6 +210,41 @@ def add_on_breadcrumb(self, on_breadcrumb) -> None:
     def remove_on_breadcrumb(self, on_breadcrumb) -> None:
         self.configuration.remove_on_breadcrumb(on_breadcrumb)
 
+    def leave_breadcrumb(
+        self,
+        message: str,
+        metadata: Dict[str, Any] = {},
+        type: BreadcrumbType = BreadcrumbType.MANUAL
+    ) -> None:
+        # Don't create breadcrumbs if the max_breadcrumbs is 0 as they would
+        # be immediately discarded anyway
+        if self.configuration.max_breadcrumbs == 0:
+            return
+
+        # Default the type to "manual" if it's not a valid BreadcrumbType
+        if not isinstance(type, BreadcrumbType):
```

Comment:
Is it reasonable to coerce string values to breadcrumb types here, to make the invocation more concise? (Did we have this conversation already, possible deja vu)

```py
Bugsnag.leave_breadcrumb("did the thing", {"button": "B"}, "navigation")
```
?

Reply:
Yeah, that makes sense to me